docs/api/scheduler.md

```diff
@@ -1705,6 +1621,9 @@ yunikorn_scheduler_vcore_nodes_usage{range="(80%,90%]"} 0
 yunikorn_scheduler_vcore_nodes_usage{range="(90%,100%]"} 0
 yunikorn_scheduler_vcore_nodes_usage{range="[0,10%]"} 0
 ```
+### Error response
+
+**Code** : `500 Internal Server Error`
```

Comment:
I didn't find the code for '/ws/v1/metrics' to return 500. Is this a misplace?

Reply:
Yep, it's a mistake. Thank you for pointed that out!